### DISCUSSION TITLE
Sync stalled that is can't seem to get past a peak number of

### DISCUSSION THREAD

**Discussion by Rich9Kr** [Chia-Network/chia-blockchain#6314] [Support]
I have already deleted one plot to see if that would make a difference in space even though I have a 5TB drive dedicated to Chia

Sync is not completing. I even deleted a 1 of my 9 plots to get where the estimated total plot size is less than network drive space even though drive space is off since I have a 5 TB drive nowhere near the 907 PiB,'

I am using a 5TB drive connected to my computer to do the mining in

Does this matter?
Total Size of Plots

912.274 GiB
Total Network Space

907.359 PiB
Best estimate over last 5 hours

Seems like it doesn't go past this point for days

Full Node Status
Status
Syncing 193536/284590
Connection Status
Connected
Network Name
mainnet
Peak Height
193536
Peak Time
04/27/2021 4:42:21 AM
Difficulty
54
VDF Sub Slot Iterations
110624768
Total Iterations
635507339856
Estimated network space
907.359 PiB" data-target-type="discussion" data-textarea-id="discussion-3376400-body" data-slash-command-url="/Chia-Network/chia-blockchain/slash_apps?subject_gid=MDEwOkRpc2N1c3Npb24zMzc2NDAw" style="box-sizing: border-box;"></div></form></div></div></div></div></div></div></div><br class="Apple-interchange-newline">

**Comment by BrandtH22** [comment]
@Rich9Kr  , if you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
